awsconfigs/common/aws-secrets-manager/base/kustomization.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: kubeflow
 resources:
-- secrets-mount.yaml
```

Comment:
The secrets-mount.yaml file is no longer necessary and can be delete since it has been replaced by the deployment.yaml file correct?

Reply:
Yes, I'll delete the file